packages/camera/lib/camera_tizen.dart

```diff
@@ -0,0 +1,580 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math';
+
+import 'package:camera_platform_interface/camera_platform_interface.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:stream_transform/stream_transform.dart';
+
+import 'src/type_conversion.dart';
+import 'src/utils.dart';
+
+const MethodChannel _channel = MethodChannel('plugins.flutter.io/camera_tizen');
+
+/// A Tizen implementation of [CameraPlatform].
+class CameraTizen extends CameraPlatform {
```

Comment:
just a question
In the implementation of camera_android, camera_android.dart exports src/android_camera.dart.
And the class name is `AndroidCamera`.

Shouldn't this modify if we followed android implementation?

Reply:
We don't have to follow their structure unless there's any benefit for doing so. This style is more consistent with other packages (such as path_provider_tizen and etc.) in our repo.